tests/broken_options_test.cpp

```diff
@@ -208,13 +208,313 @@ TEST_CASE("Usage descriptions work")
+
+        parser.usage(s, false);
+
+        auto actual = s.str();
+        std::string expected =
+            R"EXPECTED(usage: app_name [-tu] --env-opt --env-opt-2 --opt --opt_long [--opt_nos] --opt_nosd [--opt_with_d] [--some_long_named_option] --mopt [command line ...]
+
+about
+
+
+group1:
+
+some text
+
+  -o [ --opt ] arg                        some opt
+  --opt_long arg                          an option with an very very very very
+                                          very very very very very very very
+                                          very very very very very very very
+                                          very very very very very very very
+                                          very very very very very long
+                                          description
+  --opt_nos [=default value]              some opt without a short, but a
+                                          default
+  --opt_nosd arg                          some opt without short and default
+  -d [ --opt_with_d ] [=default value]    some opt with a default
+  -t [ --tog ]                            some toggle
+  -u [ --togg ]                           some other toggle
+
+group2:
+  --env-opt arg                           This is an option to set cool stuff.
+                                          Can be set using the environment
+                                          variable 'ENV_OPT'.
+  --env-opt-2 arg                         Can be set using the environment
+                                          variable 'ENV_OPT2'.
+  -m [ --mopt ] arg                       some multi opt
+  -x [ --some_long_named_option ] [=some very long default parameter for this fucking thing]
+                                          an option with an very very very very
+                                          very very very very very very very
+                                          very very very very very very very
+                                          very very very very very very very
+                                          very very very very very long
+                                          description
+)EXPECTED";
+
+        CHECK(actual.size() == expected.size());
+
+        std::size_t tmp = 0;
+        for (unsigned long i = 0; i < actual.size(); ++i)
+        {
+            if (actual[i] != expected[i])
+            {
+                std::cout << i << " " << actual[i] << " " << expected[i] << std::endl;
+                ++tmp;
+                if (tmp > 100)
+                    break;
+            }
+        }
+
+        REQUIRE(actual == expected);
+    }
+
+    SECTION("Change name of default group should work")
+    {
+        nitro::broken_options::parser parser("app_name", "about");
+
+        std::stringstream s;
+
+        parser.default_group("test arguments");
+        parser.accept_positionals(3);
+        parser.positional_name("command line");
+
+        parser.toggle("tog", "some toggle").short_name("t");
```

Comment:
So the test shall only check the name of the default group. Reduced the test to the bare minimum required to test the name